ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html

```diff
@@ -0,0 +1,52 @@
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<sly data-sly-use.logic="${'textinput.js'}"
+     data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}"
+     data-sly-use.label="${logic.labelPath}"
+     data-sly-use.shortDescription="${logic.shortDescriptionPath}"
+     data-sly-use.longDescription="${logic.longDescriptionPath}"
+     data-sly-use.questionMark="${logic.questionMarkPath}"></sly>
+<div class="cmp-adaptiveform-textinput"
+     data-sly-use.text="com.adobe.cq.forms.core.components.models.form.TextInput"
+     data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component">
+    <div data-sly-call="${label.label @componentId=component.id, labelValue=text.label.value, labelVisible=text.label.visible, bemBlock='cmp-adaptiveform-textinput'}"></div>
+    <div data-sly-call="${questionMark.questionMark @componentId=component.id, longDescription=text.description, bemBlock='cmp-adaptiveform-textinput'}"></div>
+    <textarea data-sly-test.useTextarea="${text.multiLine}"
+              aria-label="${text.label.visible && text.label.value}"
+              class="cmp-adaptiveform-textinput__widget"
+              id="${component.id}"
+              name="${text.name}"
+              readonly="${text.readOnly}"
+              required="${text.required}"
+              placeholder="${text.placeHolder}"
+              data-cmp-required="${text.constraintMessages.required}">${text.default}</textarea>
+    <input data-sly-test="${!useTextarea}"
+           class="cmp-adaptiveform-textinput__widget"
+           aria-label="${text.label.visible && text.label.value}"
+           type="text"
+           id="${component.id}"
+           name="${text.name}"
+           value="${text.default}"
+           readonly="${text.readOnly}"
+           required="${text.required}"
+           placeholder="${text.placeHolder}"
+           data-cmp-required="${text.constraintMessages.required}"/>
+    <div data-sly-call="${shortDescription.shortDescription @componentId=component.id, shortDescriptionVisible=text.shortDescriptionVisible, shortDescription=text.shortDescription, bemBlock='cmp-adaptiveform-textinput'}">
+    </div>
+    <div data-sly-call="${longDescription.longDescription @componentId=component.id, longDescription=text.description, bemBlock='cmp-adaptiveform-textinput'}"></div>
+    <sly data-sly-call="${clientlib.all @ categories=logic.clientlibs}"></sly>
```

Comment:
Can you share an HTML instance of the Field. To me it looks like a lot of redundant divs

```
<div>
   <div><label /></div>
   <div><div questionMark /></div>
   <input />
   <div><div shortDescription /></div>
   <div><div longDescription /></div>
</div>   
```

Reply:
_<div class="cmp-adaptiveform-textinput">
    <div>
  <label for="textinput-a42cb94a1a" class="cmp-adaptiveform-textinput__label">textinput1655981246470</label>
</div>
    <div>
  <div class="cmp-adaptiveform-textinput__questionmark" tabindex="0" role="button">

  </div>
</div>
    
    <input class="cmp-adaptiveform-textinput__widget" aria-label="textinput1655981246470" type="text" id="textinput-a42cb94a1a" name="textinput1655981246470">
    <div>
  <div id="textinput-a42cb94a1a-shortDescription" class="cmp-adaptiveform-textinput__shortdescription">
    <p>This is short description</p>

  </div>
</div>
    <div>
  <div aria-live="polite">
    <div id="textinput-a42cb94a1a-longDescription" class="cmp-adaptiveform-textinput__longdescription">
      <p>This is long description</p>

    </div>
  </div>
</div>

But this is only when the form author has enabled shortDescription and longDescription.

Reply:
_<div class="cmp-adaptiveform-textinput">
    <div>
  <label for="textinput-a42cb94a1a" class="cmp-adaptiveform-textinput__label">textinput1655981246470</label>
</div>
    <div>
  <div class="cmp-adaptiveform-textinput__questionmark" tabindex="0" role="button">

  </div>
</div>
    
    <input class="cmp-adaptiveform-textinput__widget" aria-label="textinput1655981246470" type="text" id="textinput-a42cb94a1a" name="textinput1655981246470">
    <div>
  <div id="textinput-a42cb94a1a-shortDescription" class="cmp-adaptiveform-textinput__shortdescription">
    <p>This is short description</p>

  </div>
</div>
    <div>
  <div aria-live="polite">
    <div id="textinput-a42cb94a1a-longDescription" class="cmp-adaptiveform-textinput__longdescription">
      <p>This is long description</p>

    </div>
  </div>
</div>

But this is only when the form author has enabled shortDescription and longDescription.